dev/devicelab/lib/framework/runner.dart

```diff
@@ -301,37 +308,23 @@ Future<ConnectionResult> _connectToRunnerIsolate(Uri vmServiceUri) async {
   }
 }
 
+Future<void> _acknowledgeTaskResultReceived({
+    required VmService service,
+    required String isolateId,
+  }) async {
+  try {
+    await service.callServiceExtension(
+      'ext.cocoonTaskResultReceived',
+      isolateId: isolateId,
+    );
+  } on RPCError {
+    // The target VM may shutdown before the response is received.
```

Comment:
Should we log? Or is that expected to happen often?

Reply:
I don't think it's worth logging since there's not really anything actionable here and we're not losing any state. This error will be thrown at the same frequency we've been seeing in https://github.com/flutter/flutter/issues/155475 as it's fundamentally the same issue, just in a location where the error can be safely ignored.